### PR TITLE
Added neat little shield to indicate publisher is verified

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

File: pkg/web_app/lib/script.dart [hunk 1]
```diff
@@ -13,7 +14,7 @@ import 'src/search.dart';
 import 'src/tabs.dart';
 
 void main() {
-  mdc.autoInit();
+  window.onContentLoaded.listen((_) => mdc.autoInit());
```
Comment (jonasfj) [inline pkg/web_app/lib/script.dart]: This didn't fix the init problem... but it the reduce the frequency.

**Comment by loic-sharma** [comment]
Just curious, what're the steps to become a verified publisher? I couldn't find any documentation on this topic. Thanks!

File: app/lib/frontend/templates/views/publisher/header.mustache [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{{! Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+<div class="detail-header">
+  <h2 class="title"><img
+          class="-pub-publisher-shield"
+          height="48" width="48"
+          title="Domain ownership verifed by pub.dev"
+          src="/static/img/verified-publisher.svg" />{{title}}</h2>
```
Comment (isoos) [inline app/lib/frontend/templates/views/publisher/header.mustache]: Let's use the hash URLs for better caching (or just add a TODO for it).

File: app/lib/frontend/templates/views/pkg/package_list.mustache [hunk 3]
```diff
@@ -16,7 +16,11 @@
         v <a href="{{& url}}">{{version}}</a>
         {{#show_dev_version}} / <a href="{{& dev_version_url}}">{{dev_version}}</a>{{/show_dev_version}}
         • updated: <span>{{last_uploaded}}</span>
-        {{#publisher_id}}• publisher: <a href="{{& publisher_url}}">{{publisher_id}}</a>{{/publisher_id}}
+        {{#publisher_id}}• <a href="{{& publisher_url}}"><img
+          class="-pub-publisher-shield"
+          height="20" width="20"
+          title="Published by a pub.dev verified publisher"
+          src="/static/img/verified-publisher-blue.svg" />{{publisher_id}}</a>{{/publisher_id}}
```
Comment (isoos) [inline app/lib/frontend/templates/views/pkg/package_list.mustache]: Let's use the hash URLs for better caching (or just add a TODO for it).

File: app/lib/frontend/templates/views/pkg/info_box.mustache [hunk 4]
```diff
@@ -5,7 +5,11 @@
 {{#publisher_id}}
 <h3 class="title">Publisher</h3>
 <p>
-  <a href="{{& publisher_link}}">{{publisher_id}}</a>
+  <a href="{{& publisher_link}}"><img
+          class="-pub-publisher-shield"
+          height="22" width="22"
+          title="Published by a pub.dev verified publisher"
+          src="/static/img/verified-publisher-blue.svg" />{{publisher_id}}</a>
```
Comment (isoos) [inline app/lib/frontend/templates/views/pkg/info_box.mustache]: Let's use the hash URLs for better caching (or just add a TODO for it).

File: app/lib/frontend/templates/views/pkg/header.mustache [hunk 5]
```diff
@@ -4,7 +4,11 @@
 
 Published <span>{{short_created}}</span>
 {{#publisher_id}}
-&bull; Publisher: <a href="{{& publisher_url}}">{{publisher_id}}</a>
+&bull; <a href="{{& publisher_url}}"><img
+          class="-pub-publisher-shield"
+          height="22" width="22"
+          title="Published by a pub.dev verified publisher"
+          src="/static/img/verified-publisher-blue.svg" />{{publisher_id}}</a>
```
Comment (isoos) [inline app/lib/frontend/templates/views/pkg/header.mustache]: Let's use the hash URLs for better caching (or just add a TODO for it).

File: app/lib/frontend/templates/views/pkg/package_list.mustache [hunk 6]
```diff
@@ -16,7 +16,11 @@
         v <a href="{{& url}}">{{version}}</a>
         {{#show_dev_version}} / <a href="{{& dev_version_url}}">{{dev_version}}</a>{{/show_dev_version}}
         • updated: <span>{{last_uploaded}}</span>
-        {{#publisher_id}}• publisher: <a href="{{& publisher_url}}">{{publisher_id}}</a>{{/publisher_id}}
+        {{#publisher_id}}• <a href="{{& publisher_url}}"><img
```
Comment (isoos) [inline app/lib/frontend/templates/views/pkg/package_list.mustache]: Also add a TODO to make this into a generic template.

Comment (jonasfj) [reply app/lib/frontend/templates/views/pkg/package_list.mustache]: How so?

**Comment by jonasfj** [comment]
@loic-sharma, hang tight, documentation will be out soon :)

**Pull request merged**